lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -1295,13 +1330,18 @@ class MongoClientInterface {
             'value.versionId': { $exists: false },
         };
 
+        const hour = 60 * 60 * 1000;
+        const cmpDate = new Date(Date.now() - hour);
```

Comment:
Can you use Date object methods to set the hours? I know that this works but that way is more clean.